### PR TITLE
Jetpack Sync: Sender: Return errors instead of plain `false`

### PR DESCRIPTION
**Pull request by lezama**
Improve debugging output for AT

### CODE REVIEW

File: sync/class.jetpack-sync-sender.php [hunk 1]
```diff
@@ -83,12 +83,13 @@ public function do_sync() {
 	public function do_sync_and_set_delays( $queue ) {
 		// don't sync if importing
 		if ( defined( 'WP_IMPORTING' ) && WP_IMPORTING ) {
-			return false;
+			return new WP_Error( 'is_importing' );;
```
Comment (dereksmart) [inline sync/class.jetpack-sync-sender.php]: extra semicolon

File: sync/class.jetpack-sync-sender.php [hunk 2]
```diff
@@ -83,12 +83,13 @@ public function do_sync() {
...
 		}
 
 		// don't sync if we are throttled
 		if ( $this->get_next_sync_time( $queue->id ) > microtime( true ) ) {
-			return false;
+			return new WP_Error( 'sync_throttled' );;
```
Comment (dereksmart) [inline sync/class.jetpack-sync-sender.php]: extra semicolon

**Comment by lamosty** [comment]
This is awesome, thanks!

**Comment by enejb** [comment]
❤️

**Pull request merged**